apps/site/src/layouts/Layout.astro

```diff
@@ -0,0 +1,35 @@
+---
+import "~/styles/global.css";
+import "@fontsource-variable/dosis";
+import "@fontsource/atkinson-hyperlegible/400.css";
+import "@fontsource/atkinson-hyperlegible/700.css";
+import Banner from "~/components/layout/Banner.astro";
+import Header from "~/components/layout/Header.astro";
+import Footer from "~/components/layout/Footer.astro";
+import Analytics from "~/components/layout/Analytics.astro";
+
+export interface Props {
+  title: string;
+}
+
+const { title } = Astro.props;
+---
+
+<html lang="en" class="h-full">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="generator" content={Astro.generator} />
+    <title>{title}</title>
+    <Analytics />
```

Comment:
As stated in [Cloudflare's docs](https://developers.cloudflare.com/analytics/web-analytics/getting-started/#sites-proxied-through-cloudflare), this script goes before the closing body tag.

> Add the JS snippet to any of your website’s HTML pages before the ending body tag.